### PR TITLE
add chat users command

### PR DESCRIPTION
**Pull request by lukas016**

### CODE REVIEW

File: app/chat/users.go [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+		if opts.Chat == "" { // defaults to me(saved messages)
+			return fmt.Errorf("missing domain id")
+		}
+
+		peer, err = utils.Telegram.GetInputPeer(ctx, manager, opts.Chat)
+		if err != nil {
+			return fmt.Errorf("failed to get peer: %w", err)
+		}
+
+		color.Yellow("WARN: Export only generates minimal JSON for tdl download, not for backup.")
```
Comment (iyear) [inline app/chat/users.go:61]: Please remove it

File: app/chat/users.go [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+		}
+		defer multierr.AppendInvoke(&rerr, multierr.Close(f))
+
+		enc := jx.NewStreamingEncoder(f, 512)
+		defer multierr.AppendInvoke(&rerr, multierr.Close(enc))
+
+		// process thread is reply type and peer is broadcast channel,
+		// so we need to set discussion group id instead of broadcast id
+		id := peer.ID()
+		if ch.IsBroadcast() {
```
Comment (iyear) [inline app/chat/users.go:108]: I think we don't need this if section. It makes sense to get the subscribers of the broadcast channel, we don't need to convert to linked group

**Comment by iyear** [comment]
Thanks for your contrib! I left some comments

File: app/chat/users.go (hunk 2 above)
Comment (lukas016) [reply app/chat/users.go]: fixed

File: app/chat/users.go (hunk 1 above)
Comment (lukas016) [reply app/chat/users.go]: fixed

**Comment by iyear** [comment]
LGTM. Thanks!

**Pull request merged**